modules/images/docs/api-reference/binary-image-api.md

```diff
@@ -17,52 +17,27 @@ The format is reported using MIME types strings. Supported binary formats and th
 const response = await fetchFile(imageUrl);
 const arrayBuffer = await response.arrayBuffer();
 
-const mimeType = getBinaryImageMIMEType(arrayBuffer);
-const {width, height} = getBinaryImageSize(arrayBuffer, mimeType);
+const metadata = getBinaryImageMetadata(arrayBuffer);
+if (medata) {
+  const {width, height, mimeType} = metadata;
+}
 ```
 
 ## Functions
 
-### isBinaryImage(imageData : ArrayBuffer [, mimeType : String]) : Boolean
-
-Parameters:
-
-- `imageData`: Binary encoded image data.
-- `mimeType`: A MIME type string.
-
-Returns `true` if the binary data represents a known binary image format or matches the supplied `mimeType`.
-
-Parameters:
-
-- `mimeType`: If supplied, checks if the image is of that type. If not supplied, returns `true` if imageData corresponds to a know supported image format.
-
-### getBinaryImageMIMEType(imageData : ArrayBuffer) : String | null
+### getBinaryImageMetadata(imageData: ArrayBuffer | DataView): object | null
 
 Parameters:
 
 - `imageData`: Binary encoded image data.
 
-Returns:
-
-- the MIME type of the image represented by the data, or `null` if it could not be identified.
-
-### getBinaryImageSize(imageData : ArrayBuffer, mimeType? : String) : Object
-
-Extracts the size of the image in `imageData`.
-
-Parameters:
-
-- `imageData`: Binary encoded image data.
-- `mimeType`: A MIME type string
-
-Returns:
-
-- an object with fields containing the size of the image represented by the data.
+Returns a metadata object describing the image. Returns `null` if the binary data does not represent a known binary image format.
```

Comment:
This conflicts with the statement below (throws if image is not in a supported binary format)